avionics/CMakeLists.txt

```diff
@@ -154,6 +153,10 @@ target_sources(testexe_LedBlinker PUBLIC common/src/tasks/led_blinker.cpp)
 ROCKET_CREATE_TEST(RepeatChecksState)
 
 ROCKET_CREATE_TEST(States)
+target_sources(testexe_States PUBLIC
```

Comment:
Seems that if all of the extra sources are commented out, this `target_sources` can be removed